Jaffenheimer/p7-internet
# Develop recipe generation endpoint + new a prompt to ChatGPT

**Features added**
- Adds recipegen endpoint to frontend 
- Adds new prompt to backend + updated the test for it
- Small ui update to fullrecipeview


**Importent before testing**
- Remember to drop your tables and rebuild or drop all recipes, because we use a new format for responses


**Testing**
- Create a recipe and check that it works
- Check tests for frontend and backend
